rest/rest-resources/src/main/java/org/apache/brooklyn/rest/transform/CatalogTransformer.java

```diff
@@ -131,7 +132,11 @@ public static CatalogItemSummary catalogItemSummary(BrooklynRestResourceUtils b,
     }
 
     public static CatalogPolicySummary catalogPolicySummary(BrooklynRestResourceUtils b, CatalogItem<? extends Policy,PolicySpec<?>> item, UriBuilder ub) {
-        Set<PolicyConfigSummary> config = ImmutableSet.of();
+        final Set<PolicyConfigSummary> config = new HashSet<>();
+        final PolicySpec<?> spec = (PolicySpec<?>) b.getCatalog().createSpec((CatalogItem) item);
```

Comment:
wrap in a `try-catch`, could throw on invalid catalog item (for example some dependency no longer exists)